money-monkey-web/drizzle.config.ts

```
@@ -5,7 +5,8 @@ export default {
   schema: "./lib/db/schema.ts",
   dialect: "postgresql",
   dbCredentials: {
-    url: process.env.POSTGRES_URL!,
+    url: process.env.DATABASE_URL!,
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)